[Bug]: netdata/devenv is broken and does not contain correct dependencies

### Bug description

```
root@f6f466fa5f15:/workspaces/netdata# cmake .
-- The C compiler identification is GNU 10.3.0
-- The CXX compiler identification is GNU 10.3.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - found
-- Found Threads: TRUE  
-- Found PkgConfig: /usr/bin/pkg-config (found version "0.29.2") 
-- building for: release
-- link time optimization: supported
-- system name: Linux
-- Checking for module 'uuid'
--   Found uuid, version 2.36.0
-- Checking for module 'zlib'
--   Found zlib, version 1.2.11
-- Checking for module 'libuv'
--   Found libuv, version 1.38.0
-- Checking for module 'liblz4'
--   Found liblz4, version 1.9.2
-- Looking for JudyLLast
-- Looking for JudyLLast - found
-- Judy library found
-- Checking for module 'openssl'
--   Found openssl, version 1.1.1f
-- Checking for module 'json-c'
--   Found json-c, version 0.15
-- Checking for module 'libipmimonitoring'
--   No package 'libipmimonitoring' found
-- Checking for module 'libmnl'
--   Found libmnl, version 1.0.4
-- Checking for module 'libnetfilter_acct'
--   No package 'libnetfilter_acct' found
-- Checking for module 'libxenstat'
--   No package 'libxenstat' found
-- Checking for module 'libcrypto'
--   Found libcrypto, version 1.1.1f
-- Checking for module 'libssl'
--   Found libssl, version 1.1.1f
-- Checking for module 'libcurl'
--   No package 'libcurl' found
-- Checking for module 'aws-cpp-sdk-core'
--   No package 'aws-cpp-sdk-core' found
-- Checking for module 'aws-cpp-sdk-kinesis'
--   No package 'aws-cpp-sdk-kinesis' found
-- Checking for module 'grpc'
--   No package 'grpc' found
-- Checking for module 'google_cloud_cpp_pubsub_protos'
--   No package 'google_cloud_cpp_pubsub_protos' found
-- Checking for module 'protobuf>=3'
--   No package 'protobuf' found
-- Checking for module 'snappy'
--   No package 'snappy' found
-- Checking for module 'libmongoc-1.0'
--   No package 'libmongoc-1.0' found
CMake Error at CMakeLists.txt:345 (file):
  file STRINGS file "/workspaces/netdata/config.h" cannot be read.


-- kinesis exporting: disabled (requires AWS SDK for C++)
-- pubsub exporting connector: disabled (requires grpc and googleapis)
-- prometheus remote write exporting: disabled (requires protobuf and snappy libraries)
-- mongodb exporting: disabled (requires mongoc library)
CMake Error at /usr/share/cmake-3.16/Modules/FindPackageHandleStandardArgs.cmake:146 (message):
  Could NOT find Protobuf (missing: Protobuf_INCLUDE_DIR)
Call Stack (most recent call first):
  /usr/share/cmake-3.16/Modules/FindPackageHandleStandardArgs.cmake:393 (_FPHSA_FAILURE_MESSAGE)
  /usr/share/cmake-3.16/Modules/FindProtobuf.cmake:624 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  CMakeLists.txt:1093 (find_package)


-- Configuring incomplete, errors occurred!
See also "/workspaces/netdata/CMakeFiles/CMakeOutput.log".
See also "/workspaces/netdata/CMakeFiles/CMakeError.log".
root@f6f466fa5f15:/workspaces/netdata# 
```

### Expected behavior

Clone netdata/netdata, open in Visual Studio Code and it works ✅

### Steps to reproduce

1. Clone netdata/netdata
2. Open repository in VSCode using the ~/.devcontainer/devcontainer.json
3. Run cmake


### Installation method

from git

### System info

```shell
n/a
```


### Netdata build info

```shell
n/a
```


### Additional info

n/a

--- a/devenv/Dockerfile
+++ b/devenv/Dockerfile
@@ -29,6 +29,10 @@ ADD https://raw.githubusercontent.com/netdata/netdata/master/packaging/installer
 
 RUN chmod +x /install-required-packages.sh && \
     /install-required-packages.sh netdata-all --non-interactive --dont-wait && \
+    apt-get install -y --no-install-recommends libprotobuf-c-dev \
+                                               libprotobuf-dev \
+                                               protobuf-c-compiler \
+                                               protobuf-compiler && \
     rm -rf /var/lib/apt/lists/* /var/cache/apt/*
 
 COPY ./bash_aliases /root/.bash_aliases

